### PR TITLE
Attempt to make import works for big vcard files

### PR DESCRIPTION
**Pull request by frankrousseau**
It shows information and add breaks between each card parsing. It results in not freezing the browser.

### CODE REVIEW

File: client/app/views/importer.coffee [hunk 1]
```diff
@@ -31,26 +34,71 @@ module.exports = class ImporterView extends BaseView
...
+    # Parse all vcards from the text file and display information about them
+    # to the user (name of each contact and number of parsed contacts).
+    # Once done, it disables the import button.
+    prepareImport: (vcardText) ->
+        # Separate all vCards to import.
+        cards = vcardText.split 'END:VCARD'
+        @toImport = []
+
+        # Prepare information displaying.
+        txt = "<p>import count: <span class=\"contact-amount\">0</span></p>"
```
Comment (aenario) [inline client/app/views/importer.coffee]: use a """ multiline string :
```
@content.html """
  <p>import count: <span class="contact-amount">0</span></p>
  <ul class="import-list></ul>
"""

File: client/app/views/importer.coffee [hunk 2]
```diff
@@ -31,26 +34,71 @@ module.exports = class ImporterView extends BaseView
...
+        txt += '</ul>'
+        @content.html txt
+
+        amount = 0
+        cards.pop()
+
+        # Recursive function that makes pause between each card import.
+        # That's useful to not block the browser while parsing the vcard
+        # file.
+        (addCard = =>
```
Comment (aenario) [inline client/app/views/importer.coffee]: You can also use the do keyword instead of ()()
```coffee
do addOneCard = => 
```

**Comment by aenario** [comment]
made a few comments, otherwise it looks good.

File: client/app/views/importer.coffee (hunk 2 above)
Comment (frankrousseau) [inline client/app/views/importer.coffee]: O great I didn't know that one!

**Comment by frankrousseau** [comment]
I forgot the translation too.

**Comment by aenario** [comment]
FYI, you dont need to escape double quotes (") inside a multi-line string (""")

**Pull request merged**